collectors/statsd.plugin/README.md

```diff
@@ -522,9 +596,10 @@ You can use it like this:
 source statsd.sh
 
 # then, at any point:
-statsd "metric1:10|g" "metric2:10|c" ...
+StatsD "metric1:10|g" "metric2:10|c" ...
 ```
-
 The function is smart enough to call `nc` just once and pass all the metrics to it. It will also automatically switch to TCP if the metrics to send are above 1000 bytes.
 
+If you have gotten thus far, make sure to check out our [Community Forums](https://community.netdata.cloud) to share your experience using Netdata with StatsD.
```

Comment:
You could potentially add a `## What's next?` section to the end of this document to help call this out a bit more.

```suggestion
If you have gotten thus far, make sure to check out our [community forums](https://community.netdata.cloud) to share your experience using Netdata with StatsD.
```